autogpt_platform/backend/backend/copilot/tools/list_team.py

```
@@ -0,0 +1,98 @@
+        self,
+        user_id: str | None,
+        session: ChatSession,
+        **kwargs,
+    ) -> ToolResponseBase:
+        if user_id is None:
+            return ErrorResponse(
+                message="Authentication required", session_id=session.session_id
+            )
+        try:
+            experts = await experts_db().list_experts(user_id, with_metrics=False)
+        except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)